### ISSUE TITLE
How to set outbound auth as “No authentication”?

### ISSUE THREAD

**Issue by RadhaR4** [Axway-API-Management-Plus/apimanager-swagger-promote#133]
I have the config file as shown below,
config file..
```json
{
   "name":"Test",
   "path":"/test",
   "state":"unpublished",
   "version":"1.0.0",
   "organization":"Development",
   "backendBasepath":"http://test.com/V1",
   "securityProfiles":[
      {
         "name":"_default",
         "isDefault":true,
         "devices":[
            {
               "type":"authPolicy",
               "name":"Invoke Policy",
               "order":1,
               "properties":{
                  "authenticationPolicy":"BasicAuth",
                  "useClientRegistry":false,
                  "subjectSelector":"authentication.subject.id",
                  "descriptionType":"original",
                  "descriptionUrl":"",
                  "descriptionMarkdown":"",
                  "description":""
               }
            }
         ]
      }
   ],
   "outboundProfiles":{
      "_default":{
         "routePolicy":"Routing",
         "faultHandlerPolicy":"Fault Handler"
      }
   }
}
```
However,the outbound authentication is not being set as "No authentication".Could you let me know what is missing in the config?
![image](https://user-images.githubusercontent.com/14121261/63629079-dfc4f680-c5c4-11e9-8262-bf07d10de4a0.png)

**Comment by cwiechmann** [comment]
@RadhaR4, the API-Manager automatically configures a No-Authentication, if no outboundProfiles is configured for an API. But in your case, you have configured one for your policies and that's why this profile is taken. 
But it doesn't contain the authenticationProfile to use with this profile. This is a scenario I have taken into account so far, but it's a valid use case.

For now (with version 1.6.1-1), you can adjust your API-Config like this:
```json
   ],
   "outboundProfiles":{
      "_default":{
         "authenticationProfile": "_default",
         "routePolicy":"Routing",
         "faultHandlerPolicy":"Fault Handler"
      }
   }
}
```
With the next release of Swagger-Promote it's checked, if the configured Default-OutboundProfile contains an authenticationProfile and if not, Swagger-Promote automatically inserts the _default.

**Comment by cwiechmann** [comment]
Has been fixed/enhanced in release 1.6.2.
